src/lib/journey/callbacks/password/validated-create-password.mock.ts

```diff
@@ -157,6 +157,43 @@ export default {
       ],
       _id: 6,
     },
+    {
+      type: CallbackType.ValidatedCreatePasswordCallback,
+      output: [
+        { name: 'echoOn', value: false },
+        {
+          name: 'policies',
+          value: {
+            policyRequirements: ['VALID_TYPE'],
+            fallbackPolicies: null,
+            name: 'password',
+            policies: [
+              {
+                policyRequirements: ['VALID_TYPE'],
+                policyId: 'valid-type',
+                params: { types: ['string'] },
+              },
+            ],
+            conditionalPolicies: null,
+          },
+        },
+        {
+          name: 'failedPolicies',
+          value: [
+            // TODO: Fix validation crash
+            '{ "policyRequirement": "LENGTH_BASED", "params": { "max-password-length": 0, "min-password-length": 8 } }',
+            '{ "policyRequirement": "CHARACTER_SET", "params": { "allow-unclassified-characters": true, "character-set-ranges": [  ], "character-sets": [ "0:0123456789", "0:ABCDEFGHIJKLMNOPQRSTUVWXYZ", "0:abcdefghijklmnopqrstuvwxyz", "0:~!@#$%^&*()-_=+[]{}|;:,.<>/?" ], "min-character-sets": 4 } }',
```

Comment:
Here's the new `0` number for the character sets. Notice the flag `min-character-sets`.